docs/content/doc/advanced/logging-documentation.en-us.md

```diff
@@ -290,9 +290,48 @@ messages. However, you could perhaps set this logger to work on `FATAL`.
 * `RECEIVERS`: Email addresses to send to.
 * `SUBJECT`: **Diagnostic message from Gitea**
 
-## Default Configuration
+## Debugging problems
 
-The default empty configuration is equivalent to:
+When submitting logs for during gitea issues it is often helpful to submit
```

Comment:
```suggestion
When submitting logs for submitting gitea issues it is often helpful to submit
```

I think the wording is more appropriate for issue submission context.

Reply:
```suggestion
When submitting logs in Gitea issues it is often helpful to submit
```